internal/identify/scene.go

```diff
@@ -210,22 +211,19 @@ func (g sceneRelationships) stashIDs(ctx context.Context) ([]models.StashID, err
 
 	for i, stashID := range stashIDs {
 		if endpoint == stashID.Endpoint {
-			// if stashID is the same, then don't set
-			if stashID.StashID == *remoteSiteID {
-				return nil, nil
-			}
-
```

Comment:
I think this is incorrect. This will result in identified scenes with no changes being updated.

Reply:
There is maybe a use case for this, which is something like “re scrape scenes last updated x time ago”, which would be a feature I think people would want until the stash-box side of #2903 is implemented.

Reply:
I think that the logic should be that the `updated_at` field should _only_ be updated if there is other changes. The aim of the field is to be able to determine if the remote data has changed since we last grabbed it. Ideally, the date would be set to the updated date on the remote end, but failing that, it should only be refreshed if we actually detect a change.